### DISCUSSION TITLE
Term Store tags used as filters

### DISCUSSION THREAD

**Discussion by KlingenbergM** [microsoft-search/pnp-modern-search#3616] [General]
I have a page library where I'm using the term store to manage some tags for the pages. This works perfectly fine for the search result webpart where I'm able to see the tags. But the filter webpart doesn't seem to be able to use them.

As far as I understand term store items are by default managed and doesn't need to be assigned to as refined string?

But it dosn't matter if I use `AutoowstaxIdRelevance` or `owstaxIdRole` the filters are empty.

It's been 48+ hours since I assigned the tags to the pages, so I'm pretty sure the crawl should have picked it up by now.

Running V4 on SharePoint Online.

**Comment by wobba** [comment]
Have you read the docs on "Enable localization" at https://microsoft-search.github.io/pnp-modern-search/usage/search-results/data-sources/#source-configuration? Any value used as filters/refiner has to be mapped to a refinable managed property.

**Reply by KlingenbergM** [reply]
yes I have read that, but as I said as far as I understand Term Store items are already crawled and managed? But perhaps that's not the same?

I mean I can display the managed metadate perfectly fine in the search results, so I assumed (perhaps wrongly) that we don't need to extra step of mapping (again) to a refinable string?

**Reply by wobba** [reply]
Not the same. Site columns auto create queryable/retrievable columns, not refinable ones.
